testing/scenario_app/lib/src/platform_view.dart

```diff
@@ -540,7 +518,7 @@ class MultiPlatformViewBackgroundForegroundScenario extends Scenario
     if (_lastLifecycleState == 'AppLifecycleState.inactive' &&
         message == 'AppLifecycleState.resumed') {
       _nextFrame = _secondFrame;
-      window.scheduleFrame();
+      view.platformDispatcher.scheduleFrame();
```

Comment:
Hm, one day we may want to schedule a frame for a single view instead of all views. Food for thought, not at all relevant to this change :)

Reply:
Yeah, this is something that comes up every now and then. Since we want a unified widget tree, frame scheduling is always gonna be global. There is some discussion on that in https://flutter.dev/go/multiple-views in the Background section.